Soccar [Physics]/src/nl/soccar/physics/listener/GameEventListener.java

```diff
@@ -0,0 +1,15 @@
+package nl.soccar.physics.listener;
+
+import nl.soccar.library.Ball;
+import nl.soccar.library.Session;
+import nl.soccar.library.enumeration.EventType;
+import nl.soccar.physics.GameEngine;
+
+/**
+ * @author PTS34A
+ */
+public interface GameEventListener {
+
+    void onBallInGoal(GameEngine engine, Session session, Ball ball, EventType type);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)